frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json

```diff
@@ -87,15 +87,15 @@
   "Time to live": "Time to live",
   "Time to live is the time that an object is stored in a caching system before it is deleted or refreshed. Default: 1 hr, Min: 15 mins, Max: 24 hrs": "Time to live is the time that an object is stored in a caching system before it is deleted or refreshed. Default: 1 hr, Min: 15 mins, Max: 24 hrs",
   "Read NamespaceStores": "Read NamespaceStores",
-  "Select list of namespace stores, defines the read targets of the namespace bucket": "Select list of namespace stores, defines the read targets of the namespace bucket",
+  "Select a list of NamespaceStore which defines the read targets of the namespaced bucket": "Select a list of NamespaceStore which defines the read targets of the namespaced bucket",
   "Create NamespaceStore": "Create NamespaceStore",
   "{{nns, number}} namespace store ": "{{nns, number}} namespace store",
   "{{nns, number}} namespace store _plural": "{{nns, number}} namespace stores",
   " selected": " selected",
   "Write NamespaceStore": "Write NamespaceStore",
-  "Select a single namespace store, defines the write target of the namespace bucket": "Select a single namespace store, defines the write target of the namespace bucket",
+  "Select a single NamesapceStore which defines the write targets of the namespaced bucket": "Select a single NamesapceStore which defines the write targets of the namespaced bucket",
   "Read and Write NamespaceStore ": "Read and Write NamespaceStore ",
-  "Select one namespace-store, defines the read and write targets of the namespace bucket": "Select one namespace-store, defines the read and write targets of the namespace bucket",
+  "Select one NamespaceStore which defines the read and write targets of the namespaced bucket": "Select one NamespaceStore which defines the read and write targets of the namespaced bucket",
```

Comment:
Select one Namespace store which defines the read and write targets of the namespaced bucket.